test/remote/gateways/remote_airwallex_test.rb

```diff
@@ -0,0 +1,140 @@
+require 'test_helper'
+
+class RemoteAirwallexTest < Test::Unit::TestCase
+  def setup
+    @gateway = AirwallexGateway.new(fixtures(:airwallex))
+
+    # https://www.airwallex.com/docs/online-payments__test-card-numbers
+    @amount = 100
+    @declined_amount = 8014
+    @credit_card = credit_card('4000100011112224')
+    @declined_card = credit_card('2223 0000 1018 1375')
+    @options = { return_url: 'https://example.com' }
```

Comment:
It seems like this field is necessary so I'm wondering if we should enforce its presence in the gateway adapter somehow? Or is there a plan to make sure that Spreedly's integration with Airwallex is always providing a default value here?

Reply:
This field is, in fact, necessary; I felt a little weird about `requires!`ing it in setup, since it's not really a credential, but maybe I should get over it. Airwallex does return a useful error that this integration then passes through, but maybe it's better for the complaint to come from the expected level (i.e., the AM API)?

Reply:
💡 Turns out I can `requires!` it in `purchase`, which is the only place it's actually needed!